src/components/download-track-progress-drawer/DownloadTrackProgressDrawer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react'
+
+import LottieView from 'lottie-react-native'
+import { Platform, StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import Drawer from 'app/components/drawer'
+import Text from 'app/components/text'
+import { useDrawer } from 'app/hooks/useDrawer'
+import {
+  getDownloadedPercentage,
+  getFileName,
+  getFetchTask
+} from 'app/store/download/selectors'
+
+const styles = StyleSheet.create({
+  view: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    paddingTop: 16,
+    paddingHorizontal: 24
+  },
+
+  title: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 21,
+    color: '#7E1BCC',
+    marginBottom: 28
+  },
+
+  fileName: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 18,
+    lineHeight: 24,
+    color: '#C2C0CC',
+    marginBottom: 12,
+    textAlign: 'center'
+  },
+
+  downloadContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginBottom: 32
+  },
+
+  loadingIcon: {
+    width: 40,
+    height: 40,
+    alignSelf: 'flex-start',
+    marginRight: 10
+  },
+
+  downloadPercent: {
+    fontFamily: 'AvenirNextLTPro-Heavy',
+    fontSize: 40,
+    color: '#858199',
+    lineHeight: 46
+  }
+})
+
+const DownloadTrackProgressDrawer = () => {
+  const [isOpen, setIsOpen] = useDrawer('DownloadTrackProgress')
+
+  const downloadPercentage = useSelector(getDownloadedPercentage)
+  const fetchTask = useSelector(getFetchTask)
+  const fileName = useSelector(getFileName)
+
+  const handleClose = useCallback(() => {
+    if (fetchTask) {
+      fetchTask.cancel()
+    }
+    setIsOpen(false)
+  }, [fetchTask, setIsOpen])
+
+  return (
+    <Drawer isOpen={isOpen} onClose={handleClose}>
+      <View style={styles.view}>
+        <Text style={styles.title}>Downloading</Text>
```

Comment:
Can put this text in a messages object? That way it'll will be easier if we support i18n in the future